DEVELOPER.md

```diff
@@ -0,0 +1,77 @@
+CURT API v3 - Setup Local Development
+=========
+---------
+This document explains the steps to setup a developer environment for the Go API project.
+> This document assumes you have already setup your local databases.
+ 
+Required Access:
+-
+- GitHub Access
+- Dealer Account API Key ([http://dealers.curtmfg.com/Index](http://dealers.curtmfg.com/Index))
```

Comment:
I think it's just dealers.curtmfg.com after going live with the new one yesterday.